src/useIsShake.ts

```diff
@@ -27,10 +29,10 @@ export function useIsShake() {
     if (newX >= 0) polls[1] = newX;
 
     let change;
-    if (polls[0] !== null && polls[1] !== null) {
+    if (polls[0] && (polls[1] || polls[1] === 0)) {
```

Comment:
Rewrote this logic to meet typeScript errors concerning possibility of `undefined` being passed to following subtraction operation in assignment of `change`